src/theater.js

```diff
@@ -103,6 +103,100 @@
       return self.utils.randomFloat(0, 1) > self.current.accuracy;
     },
 
+    parseHTML: function (string) {
+      var self = this;
+      self.current.htmlStartTags = [];
+      self.current.htmlEndTags = [];
+
+      // Do a quick scan and find out if there's any HTML in the speech
+      // If there is, we need to save the positions, lengths, and contents.
+      var htmlTag = {
+        pos: -1,
+        length: -1,
+        string: ""
+      };
+
+      for(var c = 0; c < string.length; c++) {
+        var character = string.charAt(c);
+
+        if(character === '<')
+            htmlTag.pos = c;
+
+        if(htmlTag.pos >= 0)
+            htmlTag.string += character;
+
+        if(character === '>') {
+            htmlTag.length = c - htmlTag.pos + 1;
+
+            var tagCopy = {
```

Comment:
No rush! It's your project! I just wanted to contribute because it looked like an interesting problem and I wanted to see if I could solve it. Happy to clarify or clean up things as necessary.